components/03-organisms/galleries/media-grid/_yds-media-grid.scss

```diff
@@ -0,0 +1,59 @@
+@use '../../../00-tokens/tokens';
+@use '../../grid-mixins' as grid;
+
+.media-grid {
+  @include tokens.spacing-page-section;
+}
+
+.media-grid__heading {
+  @include tokens.h2-yale-new;
+
+  margin-bottom: var(--spacing-page-inner);
+}
+
+.media-grid__items {
+  @include grid.secondary(true);
+}
+
+[data-media-grid-variation='interactive'] {
```

Comment:
To follow convention we established in other components I think this should be wrapped inside `.media-grid__item` with a parent selector? Or was this authored in this manner to avoid grid within a grid inheritance?

Reply:
Good call. I think I this happened after a refactor, and I just missed it.